### PR TITLE
Added Bootstrap Switch

### PR DESCRIPTION
**Pull request by psav**
{{pytest: cfme/tests/infrastructure/test_provisioning.py}}

### CODE REVIEW

File: cfme/web_ui/__init__.py [hunk 1]
```diff
@@ -3230,3 +3230,28 @@ def to_emails(self, emails):
 @fill.method((EmailSelectForm, tuple))
 def fill_email_select_form(form, emails):
     form.to_emails = emails
+
+
+class BootstrapSwitch(object):
+    def __init__(self, name):
+        self.name = name
+        self.locator = "//input[@id='{}']/../span[contains(@class, 'bootstrap-switch-handle-{}')]"
```
Comment (mfalesni) [inline cfme/web_ui/__init__.py]: maybe use `quoteattr` here for the id?

**Comment by dajoRH** [comment]
Lint report for commit f5b129a2ff6013ae30cc42743105e4ec48dd5e89:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 45194e7d957d743c37186891fc8f273d379372e2:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c2512ff8ceb911de0f1e60f03ad391c46d9941cd:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit dc5e35f7b4d7644929dc7ca7708a6782acba850b:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 36f603a1ac3ecb2b36ca3ddbe79b3a057dfe6c1a:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/web_ui/__init__.py [hunk 2]
```diff
@@ -3238,3 +3272,112 @@ def to_emails(self, emails):
...
+
+    def uncheck(self):
+        """Convenience function"""
+        self.fill(False)
+
+    def _el(self):
+        return sel.move_to_element(self.locator)
+
+    def is_selected(self):
+        return self._el.is_selected()
```
Comment (mfalesni) [inline cfme/web_ui/__init__.py]: Hmm, `._el` is a function, not a property. I believe you forgot to make it a property. `.is_selected` might also be a property ...

**Comment by dajoRH** [comment]
Lint report for commit cc27f0e13fac5f491cad18bd38da2f6ff14b655c:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**